### DISCUSSION TITLE
Setting exclusive_order = False

### DISCUSSION THREAD

**Discussion by alessiaf12** [kernc/backtesting.py#685] [General]
What's the rationale behind the exclusive_order parameter when it's set to False in Backtest class? How are the entry and close positions chosen?

I perfectly understand how the parameter works when set to True, however when False is set i cannot understand properly which is the rationale impllied.

**Comment by kernc** [comment]
[`Backtest(..., exclusive_orders=True)`](https://kernc.github.io/backtesting.py/doc/backtesting/backtesting.html#backtesting.backtesting.Backtest) calls `position.close()` before opening any new order.

One would leave `exclusive_orders=False` if they wanted to add to ([pyramid](https://www.investopedia.com/articles/trading/09/pyramid-trading.asp)) existing positions, e.g.
```py
    def next(self):
        assert self.position.size > 0
        self.buy(size=10)
        # On next iteration, position.size would be +10
```

**Reply by alessiaf12** [reply]
Thank you for yout reply.

But it's still not very clear to me. Can I have more information about it? maybe a more datailed example.

**Reply by kernc** [reply]
An example would be wanting to add to an existing trade. You already have 10 stocks and want to buy another 10.

But with `exclusive_orders=True`, any new call to [`Strategy.buy(size=10)`](https://kernc.github.io/backtesting.py/doc/backtesting/backtesting.html#backtesting.backtesting.Strategy.buy) would automatically close the prior trade.
